code/modules/vending/cola.dm

```diff
@@ -103,6 +103,16 @@
 		/obj/item/reagent_containers/cup/soda_cans/shamblers = 10,
 		)
 	product_slogans = "~Shake me up some of that Shambler's Juice!~"
-	product_ads = "Refreshing!;Jyrbv dv lg jfdv fw kyrk Jyrdscvi'j Alztv!;Over 1 trillion souls drank!;Thirsty? Nyp efk uizeb kyv uribevjj?;Kyv Jyrdscvi uizebj kyv ezxyk!;Drink up!;Krjkp."
+	product_ads = "Refreshing!;Thirsty for DNA? Satiate your craving!;Over 1 trillion souls drank!;Made with real DNA!;The hivemind demands your thirst!;Drink up!;Absorb your thirst."
```

Comment:
sadge

Reply:
It's still gibberish if you don't understand the language being spoken.  Also makes it sort of like an easter egg.